kitten-scientists.user.js

```diff
@@ -712,25 +734,39 @@ var run = function() {
     ReligionManager.prototype = {
         manager: undefined,
         crafts: undefined,
-        build: function (name) {
-            var build = this.getBuild(name);
-            var button = this.getBuildButton(name);
+        build: function (name, variant) {
+            var build = this.getBuild(name, variant);
+            var button = this.getBuildButton(name, variant);
 
             if (!button || !button.model.enabled) return;
 
             //need to simulate a click so the game updates everything properly
             button.domNode.click(build);
             storeForSummary(name, 1, 'faith');
-
-            activity('Kittens have discovered ' + build.label, 'ks-faith');
-        },
-        getBuild: function (name) {
-            return game.religion.getRU(name);
+			if (variant === "s") {
+				activity('Kittens have discovered ' + build.label, 'ks-faith');
+			} else {
+				activity('Kittens have built a new ' + build.label, 'ks-faith');
+			}
```

Comment:
It looks like there is extra tabs being added before the code. Align the code with the rest of the function to keep code easy to read.